src/vs/workbench/contrib/notebook/browser/view/cellParts/codeCell.ts

```diff
@@ -330,7 +330,13 @@ export class CodeCell extends Disposable {
 			this.templateData.editor?.focus();
```

Comment:
Might be more readable if the condition in `shouldUpdateDOMFocus` is inlined or the name includes "editor"